src/main/java/com/manardenza/utils/Injector.java

```diff
@@ -0,0 +1,26 @@
+package com.manardenza.utils;
+
+import com.manardenza.controller.HotelController;
+import com.manardenza.controller.UserController;
+import com.manardenza.dao.HotelDaoImpl;
+import com.manardenza.dao.UserDaoImpl;
+import com.manardenza.login.CurrentUser;
+import com.manardenza.service.HotelService;
+import com.manardenza.service.UserService;
+import lombok.Getter;
+
+import java.io.File;
+
+public class Injector {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)